wolox-books/src/app/screens/auth/screens/book-list/components/book-card/book-card.component.scss

```diff
@@ -0,0 +1,26 @@
+@import './src/scss/colors';
+
+.card {
+  background-color: $white;
+  width: 250px;
+  min-height: 322px;
+  padding: 25px;
+  display: flex;
+  flex-direction: column;
+  box-shadow: 8px 8px 6px -6px $alto;
+  margin: 15px;
+
+  .image {
+    align-self: center;
+    height: 200px;
+    width: 200px;
+    background-color: $lightgrey;
+    object-fit: cover;
```

Comment:
for this case it's better to use contain since you want to see the full book cover.